.github/workflows/main.yml

```diff
@@ -159,18 +159,34 @@ jobs:
 
   release:
     runs-on: ubuntu-latest
-    needs: [test-node, test-chrome, test-chrome-webworker, test-firefox, test-firefox-webworker, test-electron-main, test-electron-renderer, test-interop]
+    needs: [
+      test-node,
+      test-chrome,
+      test-chrome-webworker,
+      test-firefox,
+      test-firefox-webworker,
+      test-electron-main,
+      test-electron-renderer,
+      test-interop
```

Comment:
test-examples?

Reply:
Examples are tested in a separate workflow file, I don't think you can depend on jobs defined in other files yet.